docs/api-testing/use-cases/saving-token-kv.md

```diff
@@ -0,0 +1,278 @@
+
+   <img src={useBaseUrl('img/api-testing/kv-examples/token-assert.png')} alt="assert exists" />
+
+   `token_var` is available for further use inside the test (and inside your Organization).
+
+9. Add the **POST** component and then, **Save Changes**.
+
+   - Url - for example `https://m2-keyvalueexample.load2.apifortress.com/profile`
+   - Variable - for example `profilePayload`
+   - Mode - for example `json`
+
+   <img src={useBaseUrl('img/api-testing/kv-examples/token-post.png')} alt="POST request" />
+
+   This step calls the endpoint that requires the token for authenticate.
+
+10. Add the **Request Header** to the **POST**.
+
+    - Name - for example `Authorization`
+    - Value - for example `${token_var}`
+
+    <img src={useBaseUrl('img/api-testing/kv-examples/token-auth.png')} alt="Request Header" />
+
+    In the request header you pass the token saved in the Key/Value Store.
+
+The final test looks like:
+
+```yaml
+- id: kv
+  key: token
+  action: load
+  var: token_var
+- id: if
+  children:
+    - id: get
+      children: []
+      url: https://m2-loadtest.load2.apifortress.com/token
+      var: payload
+      mode: json
+    - id: set
+      var: token_var
+      mode: string
+      value: ${payload.token}
+    - id: kv
+      key: token
+      action: set
+      object: token_var
+  expression: "!token_var"
+- id: assert-exists
+  expression: token_var
+- id: post
+  children:
+    - id: header
+      name: Authorization
+      value: ${token_var}
+  url: https://m2-loadtest.load2.apifortress.com/profile
+  var: profilePayload
+  mode: json
+```
+
+### Using the Key/Value in Other Tests
+
+To use the Key/Value Store in other tests in the same Organization, you have to first load the value from the Key/Value Store and then use it.
+
+<img src={useBaseUrl('img/api-testing/kv-examples/token-kv-tests.png')} alt="Load the key/value in other tests" width="450"/>
+
+- Action - for example `Load`
+- Key - for example `token`
+- Variable - for example `my_token`
+
+`token` is the **Key** you assigned in the [first step](/#saving-a-token-in-a-keyvalue-store) of the previous example.
+`my_token` is the variable name you will use inside the test for referencing it.
+
+## Refreshing the Token
+
+The previous example shows how to save the token in the Key/Value Store but, once created, the token will be the same. In reality, tokens usually expires after a certain amount of time so you need to generate a new one. In the following example, you will see how to improve the previous example by refreshing the token.
+
+1. First, add the **Key/Value Store** component.
+
+   - Action - for example `Load`
+   - Key - for example `token`
+   - Variable - for example `token_var`
+
+   <img src={useBaseUrl('img/api-testing/kv-examples/token-kv1.png')} alt="load method in key value component" />
+
+   It loads the token: the first time you run the test, the value will be empty then, it will be the token retrieved from the call.
```

Comment:
```suggestion
   It loads the token: the first time you run the test, the value will be empty then, it will be the token retrieved from the following call.
```